optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/SolverBenchmarkFactory.java

```diff
@@ -103,7 +103,7 @@ protected void validate() {
     protected List<SolverMetric> getSolverMetrics(ProblemBenchmarksConfig config) {
         List<SolverMetric> out = new ArrayList<>();
         if (config == null) {
-            return out;
+            return List.of(SolverMetric.BEST_SCORE);
```

Comment:
I'd replace this with something like `getDefaultSolverMetrics()`, so that we can share this decision with the code below.